app/helpers/notes_helper.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
```

Comment:
and this?

Reply:
deleted all, except Application. They all were empty